packs/journals/ancestries.json

```diff
@@ -286,7 +286,7 @@
sembling leaders. Nonetheless, a community will sometimes appoint a single ghoran as an envoy to outsiders and even give them a title to make the \"animals\" more comfortable when receiving a message from the whole community.</p><h2>Alignment and religion</h2><p>Ghorans typically lack driving forces toward good or evil, though their plant-derived values can sometimes make them come across as amoral to people who are not likewise plant based. They have a slight tendency toward law, especially in their own communities. While most ghorans aren't particularly religious, spiritually oriented ones tend to worship nature by means of the Green Faith or, less commonly, Gozreh or one of the Eldest (such as the plantlike Green Mother). They rarely find interest in human-centric deities, as ghorans have distinctly different thought processes and priorities. Faith is a matter that usually doesn't change much from seed to seed and incarnation to incarnation, and that means that the newer deities, especially those ascended in the last few millennia (\"recently,\" from the ghoran perspective) have almost no foothold in ghoran communities.</p><h2>Names</h2><p>Most ghorans pick a new name with each incarnation, usually selecting a name or concept that they found pleasing in a past life. Occasionally, a ghoran might reuse a name for several incarnations in a row, especially if remaining in a community of other peoples, though each incarnation is still a distinct person.</p><h3>Sample Names</h3><p>Amsalu, Ash, Emnet, Sieri, Ooniel, Arshmarish, Velt, Delphinium, Hach, Emerald, Sable</p><h2>Ghoran Mechanics</h2><p><strong>Rarity</strong> Rare</p><p><strong>Hit Points</strong> 8</p><p><strong>Size</strong> Medium</p><p><strong>Speed</strong> 25 feet</p><p><strong>Attribute Boosts</strong> Constitution, Free</p><p><strong>Languages</strong> Common, Sylvan</p><p><strong>Additional Languages </strong>equal to your Intelligence modifier (if it's positive). Choose from Aklo, Draconic, Elven, Gnomish, Jotun, and any other languages to which you have access (such as the languages prevalent in your region)</p><p><strong>Traits </strong>Ghoran, Humanoid, Plant</p><p><strong>Senses</strong> Low-Light Vision</p><p><strong>Photosynthesis</strong> You gain nourishment from photosynthesis. You typically don't need to pay for food. If you go without sunlight for 1 week, you begin to starve. You can derive nourishment from specially formulated bottles of alchemical sunlight instead of natural sunlight, but these bottles cost 10 times as much as standard rations (40 sp).</p><h2>Ghoran Heritages</h2><p>Created with the express purpose of adapting to their environment, ghorans display several distinct physical morphs. Choose one ghoran heritage at 1st level.</p><p>@UUID[Compendium.pf2e.heritages.Item.Ancient Ash]</p><p>@UUID[Compendium.pf2e.heritages.Item.Enchanting Lily]</p><p>@UUID[Compendium.pf2e.heritages.Item.Strong Oak]</p><p>@UUID[Compendium.pf2e.heritages.Item.Thorned Rose]</p>",
```

Comment:
```suggestion
                "content": "<p>These intelligent plant people, created by a long-dead druid, possess a sort of immortality through their seeds—unless these are destroyed by external events other than merely the ravages of time.</p><p>During the height of the wars between Geb and Nex long ago, the archmage Nex asked the renegade druid Ghorus to create a plant that would feed his people even in the most inhospitable land. Ghorus did so, developing a flower that would adapt to every environment and withstand every sort of magecraft and spellworking. He succeeded too well. In the strange soil of the Impossible Lands, the flower grew and soon began to think—and then to walk and speak. These were the first ghorans. Despite their new awareness, ghorans found that many other peoples were adept at ignoring inconvenient truths, especially those involving where their next meal might come from. Bit by bit, seed by seed, ghorans grew themselves into humanoid forms, hoping to engender sympathy by mirroring the appearance of humanoids. It eventually worked, and the ghorans won citizenship and protected status in the nation of Nex. Even so, time and predation has taken its toll; the population of ghorans is small and ever dwindling.</p><p>Each ghoran is essentially an ancient spark of life that inhabits, successively, a series of plantlike bodies. Every twenty years or so, a ghoran produces a seed. Their old body withers away as their soul enters the seed, which then swiftly produces a new body. The process brings with it minor changes in personality and some loss of more distant memories, such that each new ghoran is related to their predecessor while still being a different individual. The art of creating more ghorans died with the druid Ghorus, so they're a tiny but stable minority in the Impossible Lands, dwindling bit by bit over the centuries as a few of them fall to violence and mischance.</p><p>If you want a character who's an ancient and alien soul, trying to survive and thrive in a strange and hostile world, you should play a ghoran.</p><h2>You Might...</h2><ul><li>Offhandedly mention events from thousands of years ago.</li><li>Find \"animal\" behaviors strange.</li><li>Constantly make small changes to your body.</li></ul><h2>Others Probably...</h2><ul><li>Have a hard time recognizing you after not seeing you for a while.</li><li>Are either intimidated or fascinated by how long you've lived and how much you know.</li><li>Have heard you're delicious.</li></ul><h2>Physical Description</h2><p>Roughly humanoid in size and shape, a ghoran's body is mostly made up of a fibrous plant material resembling a woody vine, with a face like an enormous, colorful flower. Ghorans have a great deal of control over the shape and structure of their bodies: with a few hours of effort, a ghoran might make themselves taller or shorter, long-limbed or barrel-chested. They have a reputation for tasting deliciously sweet and herbaceous. Most people merely find this quality a curiosity and don't attempt to take a bite, but animals in the wild will sometimes seek them out and attack them. Though they can learn to smile or laugh to fit in with other ancestries, they tend to use a different set of emotional signifiers, fluttering their face-petals to show happiness and making a rustling sound when agitated or upset. Every ghoran has a large cavity within their torso that contains a large seed. When a ghoran's body becomes too decrepit to function, they impart their memories into the seed and plant it into the ground. Within 1 to 2 months, a new ghoran body finishes growing from the seed, and the original ghoran collapses into a pile of inert organic matter. This curious method of propagation leaves a ghoran vulnerable, and many ghorans are defensive about the process among non-ghoran peoples.Being plants, ghorans have no innate understanding of gender in the human sense. Ghorans who find themselves in the company of other ancestries sometimes adopt a more masculine or feminine form, though just as many don't bother.</p><h2>Society</h2><p>Ghorans originated in Nex, and most reside as a tiny minority population in that land. In its major cities, they have built little enclaves of their own in neighborhoods featuring sweeping parks and massive trees, with the largest of these enclaves being in Quantium. While they don't possess the magical ability to enter an extradimensional home within a tree like dryads, ghorans live among the trees, typically making their homes within the neighborhoods' big parks. They are extremely skilled at building their abodes so that they don't disrupt the lives of the local flora. Visitors to a ghoran enclave might not realize initially that they've entered a home, assuming instead that they're in a recreational public space. Ghorans don't typically mind the unexpected company so long as the visitors are respectful of the park, of their homes, and of any ghorans they meet. The ghorans who live in major cities are relatively social and accepting of other ancestries, and they tend to be eager to interact, learn about others, and engage in trade. Outside of these cities, ghorans inhabit small, remote communities in the foothills of the Shattered Range and dwell in secluded forests, and they tend to be much more isolationist. Remembering well that Ghorus originally created their ancestry as food for the \"animals,\" they generally have no great desire to get to know other ancestries. However, as each new incarnation slightly shifts a ghoran's personality, they sometimes develop a wanderlust that spurs them to leave an isolated home and travel into the wider world. Often, such ghorans hold bits—or even troves—of ancient lore lost to the outside world for many years, so their journeys can sometimes lead to important new discoveries for others in the Impossible Lands and beyond.</p><p>Among themselves, ghorans cooperate easily and instinctively, with no need for a formal government. Work is apportioned by a silent consensus derived from thousands of years of experience, as everyone does what they're best at to cover necessary tasks. This silent consensus is already practiced at determining how to adjust and adapt when one ghoran falls ill or changes in a new incarnation. The addition of new ghorans traveling from afar or of members of other ancestries can sometimes throw off this natural rhythm. The new addition is an unknown quantity, so the entire community keeps tabs on the newcomers, to a potentially unsettling extent, for a short period of analysis. This practice isn't out of any distrust or desire for harm, but rather out of curiosity and a need to understand what the newcomers do well and where they need assistance. Once the ghoran community has drawn a conclusion, they adjust their actions accordingly—everyone simply begins to perform tasks differently to allow newcomers to fit in and contribute, with no official meeting or declaration. In most cases, one of the members of the community realizes they should tell non-ghoran newcomers what the community expects of them, but sometimes no one realizes there's a need to communicate. When this happens, it can leave the ghoran community and the newcomers perplexed at the others' actions and choices, as the community expects the newcomers to simply pitch in and fill the need while the newcomers wait to be told what the community expects of them.</p><p>Ghoran communities, operating on this ancient consensus, generally don't have anything resembling leaders. Nonetheless, a community will sometimes appoint a single ghoran as an envoy to outsiders and even give them a title to make the \"animals\" more comfortable when receiving a message from the whole community.</p><h2>Alignment and religion</h2><p>Ghorans typically lack driving forces toward good or evil, though their plant-derived values can sometimes make them come across as amoral to people who are not likewise plant based. They have a slight tendency toward law, especially in their own communities. While most ghorans aren't particularly religious, spiritually oriented ones tend to worship nature by means of the Green Faith or, less commonly, Gozreh or one of the Eldest (such as the plantlike Green Mother). They rarely find interest in human-centric deities, as ghorans have distinctly different thought processes and priorities. Faith is a matter that usually doesn't change much from seed to seed and incarnation to incarnation, and that means that the newer deities, especially those ascended in the last few millennia (\"recently,\" from the ghoran perspective) have almost no foothold in ghoran communities.</p><h2>Names</h2><p>Most ghorans pick a new name with each incarnation, usually selecting a name or concept that they found pleasing in a past life. Occasionally, a ghoran might reuse a name for several incarnations in a row, especially if remaining in a community of other peoples, though each incarnation is still a distinct person.</p><h3>Sample Names</h3><p>Amsalu, Ash, Emnet, Sieri, Ooniel, Arshmarish, Velt, Delphinium, Hach, Emerald, Sable</p><h2>Ghoran Mechanics</h2><p><strong>Rarity</strong> Rare</p><p><strong>Hit Points</strong> 8</p><p><strong>Size</strong> Medium</p><p><strong>Speed</strong> 25 feet</p><p><strong>Attribute Boosts</strong> Constitution, Free</p><p><strong>Languages</strong> Common, Sylvan</p><p><strong>Additional Languages</strong> equal to your Intelligence modifier (if it's positive). Choose from Aklo, Draconic, Elven, Gnomish, Jotun, and any other languages to which you have access (such as the languages prevalent in your region)</p><p><strong>Traits</strong> Ghoran, Humanoid, Plant</p><p><strong>Senses</strong> Low-Light Vision</p><p><strong>Photosynthesis</strong> You gain nourishment from photosynthesis. You typically don't need to pay for food. If you go without sunlight for 1 week, you begin to starve. You can derive nourishment from specially formulated bottles of alchemical sunlight instead of natural sunlight, but these bottles cost 10 times as much as standard rations (40 sp).</p><h2>Ghoran Heritages</h2><p>Created with the express purpose of adapting to their environment, ghorans display several distinct physical morphs. Choose one ghoran heritage at 1st level.</p><p>@UUID[Compendium.pf2e.heritages.Item.Ancient Ash]</p><p>@UUID[Compendium.pf2e.heritages.Item.Enchanting Lily]</p><p>@UUID[Compendium.pf2e.heritages.Item.Strong Oak]</p><p>@UUID[Compendium.pf2e.heritages.Item.Thorned Rose]</p>",
```